cuda_phase-retrieval_class/README.md

```diff
@@ -0,0 +1,35 @@
+# Student Project Python Bindings
+The student project investigates the performance and memory handling of Python bindings for CUDA C++ code created with pybind11.
+Contains the Python implementation of the algorithm as `git submodule`. The original repository is located at https://github.com/tuelwer/phase-retrieval.
+
+# Install
```

Comment:
You should add the requirement `conda` and an `environment.yml`, that can setup the environment automatically. See: https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file

Please write the file manual, to avoid clutter. `pybind11` needs to be installed via `conda install`. `cupy` and `numpy` can be installed via `pip`. The following post describes, how it works: https://stackoverflow.com/questions/35245401/combining-conda-environment-yml-with-pip-requirements-txt

Please add also version numbers, like `cupy>=9.0.0`